MockingbirdCli/Interface/ArgumentParser+Extensions.swift

```diff
@@ -160,6 +174,38 @@ extension ArgumentParser {
 }
 
 extension ArgumentParser.Result {
+  func hasWalkthroughOption(using argument: OptionArgument<Bool>) -> Bool {
+    return get(argument) != nil
+  }
+    
+    func getWalkthroughResult(environment: [String: String]) throws -> (project: Path, sources: [String], destination: String) {
+        
+        // Getting project path
+        print("\nEnter an Xcode project.")
+        guard let project = readLine() ?? environment["PROJECT_FILE_PATH"] else {
```

Comment:
`0.7.0` now supports project path inference. Perhaps we could pass the inferred value as a default somehow (will require refactoring though).